### PR TITLE
Add Swift Package Manager support

### PR DESCRIPTION
**Pull request by ejensen**
Adds support for using SwiftTweaks through the [Swift Package Manager](https://swift.org/package-manager/).

### CODE REVIEW

File: SwiftTweaks/TweakWindow.swift [hunk 1]
```diff
@@ -65,8 +65,7 @@ import UIKit
 
 		self.tweakStore = tweakStore
 
-		// Are we running on a Mac? If so, then we're in a simulator!
-		#if (arch(i386) || arch(x86_64))
+		#if targetEnvironment(simulator)
```
Comment (ejensen) [inline SwiftTweaks/TweakWindow.swift:68]: Added in Swift 4.1

File: SwiftTweaks/UIImage+SwiftTweaks.swift [hunk 2]
```diff
@@ -25,7 +25,11 @@ internal extension UIImage {
 	// NOTE (bryan): if we just used UIImage(named:_), we get crashes when running in other apps!
 	// (Why? Because by default, iOS searches in your app's bundle, but we need to redirect that to the bundle associated with SwiftTweaks
 	private convenience init?(inThisBundleNamed imageName: String) {
+		#if SWIFT_PACKAGE
+		self.init(named: imageName, in: Bundle.module, compatibleWith: nil)
```
Comment (ejensen) [inline SwiftTweaks/UIImage+SwiftTweaks.swift:29]: [`Bundle.module` is available within a Swift Package](https://developer.apple.com/documentation/swift_packages/bundling_resources_with_a_swift_package#3578939)

File: SwiftTweaksTests/SwiftTweaksTests.swift [hunk 3]
```diff
@@ -1,12 +0,0 @@
...
-//  SwiftTweaksTests
-//
-//  Created by Bryan Clark on 11/4/15.
-//  Copyright © 2015 Khan Academy. All rights reserved.
-//
-
-import XCTest
-@testable import SwiftTweaks
-
-class SwiftTweaksTests: XCTestCase { }
```
Comment (ejensen) [inline SwiftTweaksTests/SwiftTweaksTests.swift:12]: This type was unused

**Comment by Kaspik** [comment]
@bryanjclark Please, please! THIS needs to go in, we are trying to get rid of CocoaPods entirely. 🚀

**Pull request merged**